manifest.json

```diff
@@ -109,7 +109,15 @@
       "type" : "string",
       "enum" : [ "id", "external_id" ],
       "default" : "external_id",
-      "format" : "hidden"
+      "format" : "trait"
```

Comment:
Yes, I want us to stop customizing the `manifest.json` and loose them on every update, but I roll it back for this hotfix. We need to amp up the documents first.